### PR TITLE
Add `protect_content` to webhook responses

### PR DESCRIPTION
**Pull request by bomzheg**
# Description

Webhook responses now does not support protect content

## Type of change

Please delete options that are not relevant.

- [ ] Documentation (typos, code examples or any documentation update)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Test Configuration**:
* Operating System:  Windows 10
* Python version: 3.8

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/dispatcher/webhook.py [hunk 1]
```diff
@@ -448,6 +448,17 @@ def _global_disable_web_page_preview():
...
 
 
+class ProtectContentMixin:
+    def protect_content(self):
+        """
+        Protect content
+
+        :return:
+        """
+        setattr(self, "protect_content", True)
```
Comment (Olegt0rr) [inline aiogram/dispatcher/webhook.py:458]: Why default value is `True`?

File: aiogram/dispatcher/webhook.py [hunk 2]
```diff
@@ -448,6 +448,17 @@ def _global_disable_web_page_preview():
             return bot.disable_web_page_preview
 
 
+class ProtectContentMixin:
```
Comment (Olegt0rr) [inline aiogram/dispatcher/webhook.py:451]: Why do we need a mixin? Describe a case

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: aiogram/dispatcher/webhook.py (hunk 1 above)
Comment (bomzheg) [reply aiogram/dispatcher/webhook.py:458]: protect_content is verb describe action of protection of content. this method like https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L418 and https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L429 created for modify response by invoke method (is cleaner solution than change field of object)

it is not default value. This method change value of field.

File: aiogram/dispatcher/webhook.py (hunk 2 above)
Comment (bomzheg) [reply aiogram/dispatcher/webhook.py:451]: for consistency with other mixins like https://github.com/aiogram/aiogram/blob/dev-2.x/aiogram/dispatcher/webhook.py#L417  and https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L428
Client code can invoke method protect_content on created object of Response for protecting message content

**Review by Olegt0rr — APPROVED** [review]

**Review by evgfilim1 — APPROVED** [review]

**Comment by evgfilim1** [comment]
After reviewing, I've found that some methods aren't implemented as reply to webhook classes. This might be a subject to another PR.

Missing classes I've discovered:
- [ ] `SendAnimation`
- [ ] `SendPoll`
- [ ] `SendDice`
- [ ] ...

**Pull request merged**